packages/ingest/HttpProviderClient.js

```diff
@@ -65,13 +73,20 @@ class HttpProviderClient {
       c.on('fetchcomplete', (_, responseBuffer) => {
         const lines = responseBuffer.toString().trim().split('\n');
         lines.forEach((line) => {
-          const split = line.trim().split(pattern);
-          if (split.length === 3) {
-          // Some providers provide files with one number after the dot (".") ex (tmtdayacz8110_5.6)
-            if (split[1].match(/^(.*\.[\w\d]{1,4})\s*$/) !== null) {
-              const name = split[1].trimRight();
+          const trimmedLine = line.trim();
+          let match = matchLinksPattern.exec(trimmedLine);
+
+          while (match != null) {
```

Comment:
This package is also an option for implementing a "find all matches in a string by regex", but it didn't feel like we really needed it

https://github.com/ljharb/String.prototype.matchAll#readme